src/routes/verify/[verifyToken]/+page.svelte

```diff
@@ -33,24 +33,23 @@
 
     // Functions
     async function verifyTheToken() {
-        console.log(data.verifyToken);
         verified = new Promise(async (resolve, reject) => {
             try {
                 const response = await tryFetchingRestricted(
                     urls.get.verifyUserToken + data.verifyToken,
                 );
                 if (response.ok) {
-                    resolve(await response);
+                    resolve(response);
                     toastStore.trigger(toaster(successToast));
                     goto("/login");
-                } else if (response.status == 400) {
+                } else if (response.status === 400) {
                     toastStore.trigger(toaster(errorToast));
                     throw new Error("Couldnt be verified");
```

Comment:
Damit [das](https://github.com/solid-stack-solutions/voycar-frontend/pull/31#discussion_r1660321029) nicht passiert, aber trotzdem die Fehlermeldung angezeigt wird.
```suggestion
                   reject("Couldnt be verified");
```